bundles/mapping/mapmodule/service/map.layer.js

```diff
@@ -742,6 +742,9 @@ Oskari.clazz.define('Oskari.mapframework.service.MapLayerService',
                     desc: pResp.providers[id].desc
                 };
             });
+            providers.push({ id: 'analysis', name: Oskari.getMsg('LayerList', 'grouping.providers.analysis') });
+            providers.push({ id: 'myplaces', name: Oskari.getMsg('LayerList', 'grouping.providers.myplaces') });
+            providers.push({ id: 'userlayer', name: Oskari.getMsg('LayerList', 'grouping.providers.userlayer') });
```

Comment:
Instead of hardcoding these here, we should push these by the functionalities using them. For example in myplaces: https://github.com/oskariorg/oskari-frontend/blob/develop/bundles/mapping/mapmyplaces/domain/MyPlacesLayerModelBuilder.js#L40-L43

```
...
                    maplayerService.addLayerGroup(Oskari.clazz.create('Oskari.mapframework.domain.MaplayerGroup', group));
                }
                if (loclayer.organization) {
                    maplayerService.addDataProvider({ id: this.groupId, name: loclayer.organization });
                }
            }

            if (loclayer.organization) {
                layer.setOrganizationName(loclayer.organization);
                layer.setDataProviderId(this.groupId);
            }
            if (loclayer.inspire) {
...

Reply:
Changed so it works almost the same as groups.